### PR TITLE
#6 Add benchee and benchee_html version to version_note

### PR DESCRIPTION
**Pull request by azranel**
This solves #6 . I thought that we can place it together with Elixir and Erlang version. Let me know if you would like to make it a little tidier (wasn't sure if I should merge with `system` map in formatter).

### CODE REVIEW

**Comment by PragTob** [comment]
Hi there, 

first thanks for the contribution! :tada: 

Sorry for the longer delay, I'm not quite sure if I want them there or somewhere else :D 

For anyone wondering how it looks like, here is a screenshot:

selection_091

So reason I'm thinking about it is sure it's all versions, but while Elixir/Erlang are "important" versions that might affect the results of the benchmark, the benchee/benchee_html versions are more informative and ideally shouldn't change the result of the benchmark. Therefore, putting them together doesn't feel quite right. On the other hand, putting them there is easiest/most straight forward atm. On the other other hand, we might want to incorporate the whole `system` information one of these days and then Elixir and Erlang version would likely move somewhere else anyhow :)

------------------------

Besides that I'd probably want benchee/benchee_html be links to the repos but that's a minor detail.

--------------------------

I sort of still lean towards footer but am not quite sure. @devonestes any opinion/idea/input? :D

**Comment by devonestes** [comment]
Hmmmm. Well, I agree that incorporating the whole `system` information is probably the best course of action eventually. For now, though, without all that `system` information, I think it's best to split up the two. It feels odd to me, too, since which version of benchee you're running won't affect the results of your benchmarks. Maybe we can add a footer with this sort of info (including links to the repos/documentation)?

**Comment by PragTob** [comment]
Yeah footer with links is also what I envisioned originally in #6 ! Thanks for the input @devonestes :)

@azranel do you want to take stab at moving this into the footer? :)

**Comment by azranel** [comment]
@PragTob Sure. will move both of those values into footer :)

**Comment by azranel** [comment]
@PragTob sorry for so long time, had a lot of stuff to do for last couple of days. Anyway, it's here. Let me know if there is any issues or guidelines about the code 😄

**Comment by PragTob** [comment]
@azranel no worries, life is more important than OSS :)

File: lib/benchee/formatters/html.ex [hunk 1]
```diff
@@ -51,6 +53,12 @@ defmodule Benchee.Formatters.HTML do
   defp render_system_info(system, options \\ [visible: false]) do
     system_info(system, options)
   end
+  defp render_footer do
+    %{
+      benchee: Application.spec(:benchee, :vsn),
+      benchee_html: Application.spec(:benchee_html, :vsn)
+    } |> footer()
```
Comment (PragTob) [inline lib/benchee/formatters/html.ex]: I think we should rather do `footer(%{})` - a pipe with just one argument is a bit overpowered... I should maybe add credo to the build here :D

File: lib/benchee/formatters/html.ex [hunk 2]
```diff
@@ -51,6 +53,12 @@ defmodule Benchee.Formatters.HTML do
   defp render_system_info(system, options \\ [visible: false]) do
     system_info(system, options)
   end
+  defp render_footer do
```
Comment (PragTob) [inline lib/benchee/formatters/html.ex]: usually we just try to put together the same methods (although the new elixir formatter will also do that differntly)

Comment (azranel) [reply lib/benchee/formatters/html.ex]: @PragTob not sure if I understand correctly - what do you mean by "same" methods? Just above `render_footer` there are similar methods so it looks like good place to me. Could you elaborate on that?

Comment (PragTob) [reply lib/benchee/formatters/html.ex]: Oh yeah we only group the same method - as in same method and same arity. So:

```
def a do
  ...
end
def a do
  ...
end

def b do
 ...
end
def b do
  ...
end

def c do
  ...
end
```

But I think the new formatter even switches it up that all are separated by blank lines.

Sorry for the long turnaround, lots of stuff going on :)

**Comment by azranel** [comment]
@PragTob done :)

**Comment by PragTob** [comment]
Thanks a bunch :tada:

**Pull request merged**